assets/src/components/customers/CustomerUtils.ts

```diff
@@ -0,0 +1,15 @@
+import * as API from '../../api';
```

Comment:
nit -- for the file name, we've been using the convention of calling util files like this `customers/support.ts` (just a convention we used at stripe which i've kept)

Reply:
Ooo ok. I dig it. Will update!